lib/simulator-xcode-6.js

```diff
@@ -185,6 +185,34 @@ class SimulatorXcode6 extends EventEmitter {
     return appDirs.length !== 0;
   }
 
+  /**
+   * Returns user installed bundle ids which has 'bundleName' in their Info.Plist as 'CFBundleName'
+   * @param {string} bundleId - The bundle id of the application to be checked.
+   * @return {array<string>} - The list of bundle ids which have 'bundleName'
+   */
+  async getUserInstalledBundleIdsByBundleName (bundleName) {
```

Comment:
the name of this method and parameter types should match with the one we use for the real device in XCUITest driver, so we could polymorph them